src/stan/io/reader.hpp

```diff
@@ -2,9 +2,9 @@
 #define STAN__IO__READER_HPP
 
 #include <boost/throw_exception.hpp>
+#include <stan/math/prim/scal.hpp>
```

Comment:
can you revert this change? Unless there's a specific reason this needs to be included in this order, but even so, this looks like the wrong order.